runelite-api/src/main/java/net/runelite/api/ScriptID.java

```diff
@@ -408,4 +408,7 @@
 
 	@ScriptArguments(integer = 7)
 	public static final int ORBS_UPDATE_RUNENERGY = 447;
+
+	@ScriptArguments()
+	public static final int AGI_LAPS_CHECK = 418;
```

Comment:
This is absolutely not the name of this script. https://github.com/Joshua-F/cs2-scripts/blob/master/scripts/%5Bclientscript%2Clongscroll_setup%5D.cs2